rts/lra/lra-annotation-checker/src/test/java/io/narayana/lra/checker/LraCdiCheckIT.java

```diff
@@ -138,7 +138,7 @@ private void initWeld(Class<?> beanClassToCheck) {
     private void assertFailureCatalogContains(Class<?> beanClass, String failureStringToCheck) {
         Assert.assertFalse("Failure on checking bean " + beanClass.getName() + " should happen",
                 FailureCatalog.INSTANCE.isEmpty());
-        Assert.assertThat(FailureCatalog.INSTANCE.formatCatalogContent().split("\n"),
+        Assert.assertThat(FailureCatalog.INSTANCE.formatCatalogContent().split(String.format("%n")),
```

Comment:
How come you didn't use System.lineSeparator()?

Reply:
I think it resolves in the same way. I will use `System.lineSeparator()`, not a problem. Thanks for pointing that.